src/tyr/route_serializer_osrm.cc

```diff
@@ -429,6 +429,54 @@ struct IntersectionEdges {
   }
 };
 
+// Process 'indications' array - add indications from left to right
+json::ArrayPtr lane_indications(std::unique_ptr<valhalla::odin::EnhancedTripLeg_Edge>& prev_edge,
+                                const valhalla::TurnLane* turn_lane) {
```

Comment:
nit: why pass this by pointer? indeed why pass it at all? in the end you just pull the mask off of it, perhaps its best to just pass the mask to this function by value eg:
```suggestion
                                uint16_t mask) {
```

Reply:
Fair point! I made it even simpler by changing the signature to:

```CPP
lane_indications(const bool drive_on_right, const uint16_t mask)
```